### DISCUSSION TITLE
Display input image (and any additional contextual information) in Individual Predictions

### DISCUSSION THREAD

**Discussion by jordanpn** [oegedijk/explainerdashboard#171] [Ideas]
First of all, fantastic work on this project and thank you for your efforts! I find that this tool would be even more useful if one could visualize input images in the Individual Predictions tab (e.g. pictures of cats and dogs) along with any additional contextual information (beyond model features) that might be available for each prediction item. Using this tool to identify qualitative differences or perhaps labeling errors would be tremendously useful, imo.

**Comment by oegedijk** [comment]
Hi @jordanpn, the dashboard has been developed mostly with tabular data in mind,  not so much with image recognition or NLP tasks.

Are you using it now for an image recognition task? If so, how?

**Reply by jordanpn** [reply]
Yes, I am using the toolkit for evaluating an image classification model. I am particularly interested in seeing the image in the failure cases to help with debugging and understanding trends why certain images are more challenging to classify. I envisioned that images (or links to images) could be optionally passed to the dashboard much like labels and then displayed in the Individual Predictions tab.
